### DISCUSSION TITLE
How can i use external ETCD/minio storage?  Where should I changes config files?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#22959] [Q&A and General discussion] [answered]
How can i use external ETCD/minio storage?  Where should I changes config files?

**Comment by yhmo** [comment, marked as answer]
https://github.com/milvus-io/milvus/discussions/6980
https://github.com/milvus-io/milvus-helm/tree/master/charts/milvus#configuration
